Add documentation for `textHaloColor` (#437)
ZeLonewolf/openstreetmap-americana: CONTRIBUTING.md

- **`notext`** – By default, a relation missing a `ref` tag will not appear as a shield. Set this property to `true` to display a shield even if it has no `ref`. This is appropriate for one-off shield networks, which are common for toll roads and touristic routes.
- **`padding`** – An object that specifies the amount of padding on each side of the inscribed text relative to the background image.
- **`textColor`** – The color of the inscribed text to superimpose on the background.

> - **`textHaloColor`** – The color of the stroke surrounding the inscribed text to superimpose on the background.

Reviewer: Let's not overcomplicate things :-D

```suggestion
- **`textHaloColor`** – The color of the halo surrounding the inscribed text.
```
